src/zope/app/apidoc/utilities.py

```diff
@@ -238,7 +238,11 @@ def _simpleGetFunctionSignature(func, ignore_self=False):
     _checkFunctionType(func)
 
     try:
-        args, varargs, varkw, defaults = inspect.getargspec(func)
+        if six.PY3:
+            (args, varargs, varkw, defaults, kwonlyargs, kwonlydefaults,
+             annotations) = inspect.getfullargspec(func)
```

Comment:
If this is used only on Python 2, then we don't need to make it work on Python 3.

Reply:
See https://github.com/zopefoundation/zope.app.apidoc/pull/38/commits/61de1109be0e8a45ea6dbc2a64f1d4bf87e47940